extension/json/json_functions/copy_json.cpp

```diff
@@ -59,14 +59,14 @@ static BoundStatement CopyToJSONPlan(Binder &binder, CopyStatement &stmt) {
 
 	// Bind the select statement of the original to resolve the types
 	auto dummy_binder = Binder::CreateBinder(binder.context, &binder);
-	auto bound_original = dummy_binder->Bind(*stmt.select_statement);
+	auto bound_original = dummy_binder->Bind(*stmt.info->select_statement);
```

Comment:
I think this might be to blame for <https://github.com/duckdb/duckdb/actions/runs/8916754832/job/24491338899>

I'm not entirely sure but this might be binding the expressions in the original info and that gets serialized later?

Reply:
What this PR changed is that the `select_statement` is now stored in the `CopyInfo`, which get's serialized
Perhaps this was not a problem before even though it already bound the original expressions, since CopyStatement does not get serialized

Reply:
```c++
void LogicalCopyToFile::Serialize(Serializer &serializer) const {
	LogicalOperator::Serialize(serializer);
	serializer.WriteProperty(200, "file_path", file_path);
	serializer.WriteProperty(201, "use_tmp_file", use_tmp_file);
	serializer.WriteProperty(202, "filename_pattern", filename_pattern);
	serializer.WriteProperty(203, "overwrite_or_ignore", overwrite_or_ignore);
	serializer.WriteProperty(204, "per_thread_output", per_thread_output);
	serializer.WriteProperty(205, "partition_output", partition_output);
	serializer.WriteProperty(206, "partition_columns", partition_columns);
	serializer.WriteProperty(207, "names", names);
	serializer.WriteProperty(208, "expected_types", expected_types);
	serializer.WriteProperty(209, "copy_info", copy_info);
```
^ copy_info

Reply:
I can trigger an internal exception with this:
```c++
	auto string = copy_info->ToString();
	serializer.WriteProperty(209, "copy_info", copy_info);
```

So I think I'm on the right path

Reply:
Yea it also triggers a SerializationException normally, but that seems to be getting swallowed